barretenberg/cpp/src/barretenberg/execution_trace/execution_trace.cpp

```diff
@@ -140,28 +142,17 @@ void ExecutionTrace_<Flavor>::add_ecc_op_wires_to_proving_key(Builder& builder,
                                                               typename Flavor::ProvingKey& proving_key)
     requires IsGoblinFlavor<Flavor>
 {
-    // Initialize the ecc op wire polynomials to zero on the whole domain
```

Comment:
Is this still done?

Reply:
All polys are initialized to zero by default in the ProverPolynomials constructor. This was always the case (it just used to be in the Pkey constructor)